com.playeveryware.eos/Runtime/Core/SystemDynamicLibrary.cs

```diff
@@ -102,6 +102,8 @@ public partial class SystemDynamicLibrary
         "DynamicLibraryLoaderHelper_Android";
 #elif UNITY_STANDALONE_OSX
         "libDynamicLibraryLoaderHelper";
+#elif UNITY_STANDALONE_LINUX
+        "libDynamicLibraryLoaderHelper";
```

Comment:
Minor: Reduce repeated code
```suggestion
#elif UNITY_STANDALONE_OSX || UNITY_STANDALONE_LINUX
        "libDynamicLibraryLoaderHelper";
```

Reply:
Implemented suggested change